tasks/section_1/cis_1.2.x.yml

```diff
@@ -72,7 +72,7 @@
       - name: "1.2.3 | PATCH | Ensure repo_gpgcheck is globally activated | Update yum.repos"
         ansible.builtin.replace:
             name: "{{ item.path }}"
-            regexp: "^repo_gpgcheck=0"
+            regexp: ^repo_gpgcheck\s*=\s*0"
```

Comment:
There appears to be a rogue quote here, at the very end of line 75: `regexp: ^repo_gpgcheck\s*=\s*0"`